test-case/check-signal-stop-start.sh

```diff
@@ -16,9 +16,12 @@
 ##    no errors for aplay/arecord
 ##
 
-source $(dirname ${BASH_SOURCE[0]})/../case-lib/lib.sh
+set -e
 
-OPT_NAME['t']='tplg'     OPT_DESC['t']='tplg file, default value is env TPLG: $TPLG'
+# shellcheck source=case-lib/lib.sh
+source "$(dirname "${BASH_SOURCE[0]}")"/../case-lib/lib.sh
+
+OPT_NAME['t']='tplg'     OPT_DESC['t']='tplg file, default value is env TPLG'
```

Comment:
So this no longer shows the value of $TPLG?

Reply:
> So this no longer shows the value of $TPLG?

It does; I just "unresolved" the comment above that explains this. Try `-h`.